client/a8c-for-agencies/sections/sites/sites-dataviews/style.scss

```diff
@@ -21,10 +21,6 @@
 		color: var(--color-jetpack-50);
 	}
 
-	.dataviews-view-table__actions-column {
```

Comment:
I wonder if this style (and the other two in a8c-for-agencies & components/dataviews) is affecting any other screen.